java/onnxruntime/src/main/java/ml/microsoft/onnxruntime/Env.java

```diff
@@ -0,0 +1,18 @@
+package ml.microsoft.onnxruntime;
+
+public class Env {
+  static {
+    System.loadLibrary("onnxruntime-jni");
+  }
+  public Env(LoggingLevel loggingLevel, String logId) throws OrtException {
+    initHandle(loggingLevel, logId);
+  }
+  @Override
+  protected void finalize() throws Throwable {
+    dispose();
+    super.finalize();
+  }
```

Comment:
I doubt that we ever need this. Perhaps to detect that this was not AutoClosed but otherwise seems useless.

Reply:
Hi @yuslepukhin , Currently we don't. But if we want to add a global thread pool instance in the Env class, then this is a must.